src/main/resources/swagger/api-docs.yaml

```diff
@@ -1888,6 +2056,48 @@ definitions:
         type: string
         description: the role of the current user in the project
 
+  CallMetadata:
+    description: Stuff you need to know about calls
+    properties:
+      inputs:
+        type: object
+        description: map from input names onto resolved values
+      executionStatus:
+        type: string
+        description: status of call
+      backendStatus:
+        type: string
+        description: status of the backend
+      backendLogs:
+        type: object
+        description: map of log files from backend
+      outputs:
+        type: object
+        description: map from output names to local files
+      start:
+        type: string
+        format: date-time
+        description: starting time of call
+      end:
+        type: string
+        format: date-time
+        description: ending time of call
+      jobId:
+        type: string
+        description: unique ID of job
+      returnCode:
+        type: integer
+        description: result code
+      backend:
+        type: string
+        description: type of backend executing the call
+      stdout:
+        type: string
+        description: location of stdout
+      stderr:
+        type: string
+        description: location of stderr
+
```

Comment:
good catch on this! The executionEvents was added just recently and I missed it (broadinstitute/rawls#567). I added shardIndex too, and will create a PR in rawls to update it there.